app/build.gradle

```diff
@@ -35,10 +35,13 @@ android {
     }
     buildTypes {
         debug {
-
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
This is probably a bad idea, because proguard slows down builds a lot.

Reply:
This does not cause any performance issues for me. But when using android instant run (the play button of Android Studio), this causes some random crashs due to missing classes (which don't occur when generating a debuggable APK and installing that one). On the other hand, it's better to test with proguard to find bugs caused by it, but as it works now, I think it's better to disable it.

Reply:
We should definitely test a release build of the app before every release. But I don't think it's a good idea to do that every time you debug something.